pipeline/types/notification.py

```diff
@@ -1,22 +1,34 @@
-from dataclasses import dataclass
+import datetime
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import List
 
 from pipeline.types.chat import ChatData
 from pipeline.types.match import Match
+from pipeline.types.user import User
+
+
+class NotificationType(Enum):
+    # An enum representation of all the notification types devised so far; subject to change if more types are added
+    CHECK_MATCH = "New_match"
+    CHECK_MSG = "New_message"
```

Comment:
Spell out the whole word MESSAGE here, no need to abbreviate.

Also, can you make the enum strings match the name of the enum value, but lowercase? So CHECK_MESSAGE would be "check_message"